### PR TITLE
lightning: record network usage

### PR DESCRIPTION
**Pull request by buchuitoudegou**
### What problem does this PR solve?

Issue Number: ref #39674 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/lightning/config/config.go [hunk 1]
```diff
@@ -1185,7 +1188,7 @@ func (cfg *Config) CheckAndAdjustSecurity() error {
 			len(cfg.TiDB.Security.CertPath) > 0 || len(cfg.TiDB.Security.CertBytes) > 0 ||
 			len(cfg.TiDB.Security.KeyPath) > 0 || len(cfg.TiDB.Security.KeyBytes) > 0 {
 			if cfg.TiDB.Security.TLSConfigName == "" {
-				cfg.TiDB.Security.TLSConfigName = uuid.NewString() // adjust this the default value
+				cfg.TiDB.Security.TLSConfigName = uuid.New().String() // adjust this the default value
```
Comment (lance6716) [inline br/pkg/lightning/config/config.go:1191]: is this needed?

File: br/pkg/lightning/lightning.go [hunk 2]
```diff
@@ -370,6 +374,36 @@ func (l *Lightning) RunOnceWithOptions(taskCtx context.Context, taskCfg *config.
...
+		taskCfg.TiDB.UUID = uuid.New().String()
+		go func() {
+			for {
+				time.Sleep(time.Millisecond * 10)
+				log.L().Info("IOTotalBytes", zap.Uint64("IOTotalBytes", taskCfg.TiDB.IOTotalBytes.Load()))
+			}
+		}()
+	})
+	if taskCfg.TiDB.IOTotalBytes != nil {
+		o.logger.Info("not nil io total bytes")
```
Comment (lance6716) [inline br/pkg/lightning/lightning.go:389]:
```suggestion
		o.logger.Info("found IO total bytes counter")
```

File: br/pkg/lightning/lightning.go [hunk 3]
```diff
@@ -370,6 +374,36 @@ func (l *Lightning) RunOnceWithOptions(taskCtx context.Context, taskCfg *config.
...
+		mysql.RegisterDialContext(taskCfg.TiDB.UUID, func(ctx context.Context, addr string) (net.Conn, error) {
+			o.logger.Info("io connection")
```
Comment (lance6716) [inline br/pkg/lightning/lightning.go:391]:
```suggestion
			o.logger.Debug("connection with IO bytes counter")
```

File: errors.toml [hunk 4]
```diff
@@ -1376,16 +1376,6 @@ error = '''
 Ingest failed: %s
 '''
 
-["domain:8027"]
```
Comment (lance6716) [inline errors.toml:1379]: :thinking: causes a CI fail

File: br/pkg/lightning/config/config.go (hunk 1 above)
Comment (buchuitoudegou) [reply br/pkg/lightning/config/config.go:1191]: I don't understand why when I use uuid.NewString() it will report an error...

**Review by lance6716 — APPROVED** [review]

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**